### PR TITLE
Alpabetic Ordering Activity level-1

### PR DESCRIPTION
**Pull request by Anumittal**
An exercise activity of the type alphabets ordering ( https://phabricator.kde.org/T1549 ). Added level-1

### CODE REVIEW

File: src/activities/alphabeticOrder/ActivityInfo.qml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/* GCompris - ActivityInfo.qml
+ *
+ * Copyright (C) 2016 Your Name <yy@zz.org>
```
Comment (petitlapin) [inline src/activities/alphabeticOrder/ActivityInfo.qml]: Copyright

File: src/activities/alphabeticOrder/ActivityInfo.qml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ */
+import GCompris 1.0
+
+ActivityInfo {
+  name: "alphabeticOrder/AlphabeticOrder.qml"
+  difficulty: 1
+  icon: "alphabeticOrder/alphabeticOrder.svg"
+  author: "Anu Mittal &lt;anu22mittal@gmail.com &gt;"
+  demo: true
+  title: "AlphabeticOrder activity"
```
Comment (petitlapin) [inline src/activities/alphabeticOrder/ActivityInfo.qml]: qsTr() for translatable strings

File: src/activities/alphabeticOrder/AlphabeticOrder.qml [hunk 3]
```diff
@@ -0,0 +1,33 @@
+/* GCompris - alphabeticOrder.qml
+ *
+ * Copyright (C) 2016 YOUR NAME <xx@yy.org>
+ *
+ * Authors:
+ *   <THE GTK VERSION AUTHOR> (GTK+ version)
```
Comment (petitlapin) [inline src/activities/alphabeticOrder/AlphabeticOrder.qml]: copyright

File: src/activities/alphabeticOrder/resource/board/board1_0.qml [hunk 4]
```diff
@@ -0,0 +1,68 @@
+/* GCompris
+ *
+ * Copyright (C) 2015 Bruno Coudoin <bruno.coudoin@gcompris.net>
+ *
+ */
+import QtQuick 2.0
+
+QtObject {
+   property string instruction: qsTr("Easy-01")
```
Comment (petitlapin) [inline src/activities/alphabeticOrder/resource/board/board1_0.qml]: I think it should be a complete sentence here

**Comment by petitlapin** [comment]
Hi,

what is the use of the sounds?

I'm not sure if babymatch is a good idea to present this activity, we probably can do a more complete activity with all the alphabet?

Johnny

**Comment by bdoin** [comment]
I agree with Johnny, babymatch is puzzle orientated and implies many
limitation. For example it means having to create an image for each
language sign which will be unsustainable. We really want GCompris to
work in any language.

**Comment by Anumittal** [comment]
Yes, I can get your point. I was just trying my hands on Gcompris by
creating an exercise activity. And I used babymatch activity as I thought
alphabet ordering activity can be similar to the 'chronos' activity which
arranges things in particular orderwhich in turn used babymatch activity.
If you can give an idea of how can I add a simple level in it according to
the requirement, I can definitely try to build one.

On Mon, Apr 4, 2016 at 4:51 PM, Bruno Coudoin <notifications@github.com>
wrote:

>
> I agree with Johnny, babymatch is puzzle orientated and implies many
> limitation. For example it means having to create an image for each
> language sign which will be unsustainable. We really want GCompris to
> work in any language.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/bdoin/GCompris-qt/pull/132#issuecomment-205249817>
>



-- 
Regards,
Anu.

**Comment by bdoin** [comment]
Not sure to understand your question but you cannot do it with babymatch.

**Pull request closed without merging**